FindYourFriends/screens/MapScreen.js

```diff
@@ -12,8 +12,47 @@ import Card from "../components/Card";
 import colors from "../constants/colors";
 import Input from "../components/Input";
 import facade from "../facade";
+import useMutation from "@apollo/react-hooks";
 
 const MapScreen = (props) => {
+
+  const [updatePosition, { loading, error, data, called }] = useMutation(
+    facade.UPDATE_POSITION
+  );
+
+  // Example of how to use the hook
+  async function updateMyPosition(){
```

Comment:
```suggestion
  async function updateMyPosition(lon, lat){
```

Reply:
I just made this code as an example - it's not meant to be used; it's meant to show how to use the hook

Reply:
Ahh okay.